tests/libs/test_typing_extended.py

```diff
@@ -0,0 +1,39 @@
+from typing import (
```

Comment:
@rudolfix @sh-rp where should I put this test file because it doesn't fit with minimum test suite run profile I created this here?

Reply:
why can't it fit in test_typing.py? just declare pydantic UUID4 like type yourself to drop the dependency!